### PR TITLE
Add logo and benchmarks to README

### PR DESCRIPTION
**Pull request by eriknw**
This is a companion PR to https://github.com/python-graphblas/python-graphblas/pull/432

I copied the logo from that PR and manually modified it.

### CODE REVIEW

**Review by SultanOrazbayev — APPROVED** [review]
This one also looks great, Erik! :)

File: docs/_static/img/logo-name-medium.svg
[the remark this answers is not in the document]
Comment (eriknw) [reply docs/_static/img/logo-name-medium.svg:1]: Really? I don't think I see that. Can you take a screenshot of what you are seeing?

[the remark this answers is not in the document]
Comment (SultanOrazbayev) [reply docs/_static/img/logo-name-medium.svg:1]: Screenshot 2023-04-18 at 09 11 26

I checked Chrome and Safari, both look like this for me...

[the remark this answers is not in the document]
Comment (SultanOrazbayev) [reply docs/_static/img/logo-name-medium.svg:1]: And now that I'm looking at it again, the last s in algorithms is also gone... Interesting.

[the remark this answers is not in the document]
Comment (SultanOrazbayev) [reply docs/_static/img/logo-name-medium.svg:1]: I tried playing around with zoom to see if that would make a diff, but nope.

[the remark this answers is not in the document]
Comment (eriknw) [reply docs/_static/img/logo-name-medium.svg:1]: I see this now (on my macbook); it renders fine for me on Windows where I created it. Thanks for letting me know!

[the remark this answers is not in the document]
Comment (eriknw) [reply docs/_static/img/logo-name-medium.svg:1]: Okay, I think the logo should be fixed now if you'd like to take a look to verify.

**Comment by SultanOrazbayev** [comment]
nope looks the same

**Comment by eriknw** [comment]
> nope looks the same

Dope! Thanks for checking. Jim and I have learned that the font looks very different on different systems (Windows, Mac, Android, etc). So, I think @jim22k is going to convert all the svg logos to PNGs (Jim, don't forget to minimize the PNGs--such as from https://compresspng.com/ and https://tinypng.com/ --to improve page load times!).

File: docs/_static/img/logo-name-medium.svg
[the remark this answers is not in the document]
Comment (SultanOrazbayev) [reply docs/_static/img/logo-name-medium.svg:1]: To recap testing on my end: looks good on MacOS and Windows, looks truncated on Ubuntu.

**Comment by jim22k** [comment]
I switched from svg to png. This bumps the logo size from 1k to 35k, but I think it's worth it to avoid the need to grab fonts from the local machine, which leads to different looks and widths. PNG will always be consistent.

**Review by SultanOrazbayev — COMMENTED** [review]
Looks good! Interesting the file size is so large (35K), but small enough to not be an issue.

**Comment by eriknw** [comment]
Thanks for converting the svg font to paths @jim22k. I minified the svg (that you converted the font into paths) with https://vecta.io/nano and deleted the png, b/c the svg looks better.

I think this is good to go in :tada:

Now let's do the same for the svgs in `python-graphblas` :)

**Pull request merged**